scripts/userstyles.yml

```diff
@@ -102,6 +102,9 @@ collaborators:
   - &rooot
     name: rooot
     url: https://github.com/RoootTheFox
+  - &itzTheMeow_
```

Comment:
Extra underscore? Not sure why that's here...
```suggestion
  - &itzTheMeow
```

Reply:
oops sorry, used to typing the _